internal/errors/errors.go

```diff
@@ -193,6 +193,8 @@ var (
 	// ErrGatewayTimeout is returned when a gateway operation times out.
 	ErrGatewayTimeout = errors.New("gateway timeout")
 
+	ErrGatewayTimeoutForIndexDDL = errors.New("timeout waiting for response for index DDL")
```

Comment:
is this shown to the customer? what actions should they take and why does it happen from their PoV? How can they get this info?

Reply:
adding a godoc will help

Reply:
addressed in https://github.com/couchbasecloud/terraform-provider-couchbase-capella/pull/357/commits/dad2d409cae1b85424616c8fe15526c269be11e0